definitions/EiffelMetaProperty/3.1.0.yml

```diff
@@ -0,0 +1,194 @@
+$schema: http://json-schema.org/draft-04/schema#
```

Comment:
The last change I made to the PR where I introduced the YAML files was to add a copyright statement to these files, so please include it in this one too. You can just copy the one from EiffelMetaProperty/3.0.0.yml.

Reply:
There was something wrong with the merge and the latest files from master were not used. Fixed now.